src/libos/src/entry.rs

```diff
@@ -87,6 +95,23 @@ pub extern "C" fn occlum_ecall_init(log_level: *const c_char, instance_dir: *con
         unsafe { backtrace::enable_backtrace(&ENCLAVE_PATH, PrintFormat::Short) };
     });
 
+    // Parse host resolv.conf file
+    if let Err(e) = parse_resolv_conf(resolv_conf_ptr) {
+        eprintln!("failed to parse host /etc/resolv.conf: {}", e);
+    }
+
+    match parse_resolv_conf(resolv_conf_ptr) {
+        Err(e) => {
+            eprintln!("failed to parse host /etc/resolv.conf: {}", e);
```

Comment:
Use "debug!" instead of "eprintln!"

Reply:
Done, use "error" to replace "eprintln".